src/main/java/WorkShop01.kt

```diff
@@ -41,21 +52,29 @@ object WorkShop01 {
     // PS2. Оператор, который определяет остаток от деления - %
     // /
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P5Test
-    fun evenOddNumber(number: Int) {
-        TODO()
+    fun evenOddNumber(number: Int): String {
+        return if (number % 2 == 0) {
+            "Чётное"
+        } else "Нечётное"
     }
 
     //TODO 6. Необходимо реализовать лямбда-функцию и задать ей имя lambda, которая
     // 1. Принимает два значения типа Int и возвращает значение типа Int
     // 2. Результат выполнения лямбда-функции - сравнить два значения и вернуть максимальное из 2-х
     // Для проверки задания необходимо запустить тесты в классе WorkShop01P6Test
-    //val lambda: TODO()
+    val lambda: (Int, Int) -> Int = {num1, num2 ->
+        if (num1 > num2) num1 else num2
+    }
 
     // TODO 7. Необходимо реализовать функцию высшего порядка c именем compareTwoInts, которая
     //  1. Принимает на вход два числа и лямбда-функцию
     //  2. Возвращает результат исполнения лямбда-функции, в которую должны быть переданы два числа,
     //  которые функция высшего порядка получила на вход
     //  Для проверки задания необходимо запустить тесты в классе WorkShop01P7Test
+    fun compareTwoInts(num3: Int, num4: Int, lambda: (Int, Int) -> Boolean): Boolean {
```

Comment:
```suggestion
    fun compareTwoInts(num3: Int, num4: Int, lambda: (Int, Int) -> Boolean): Boolean = lambda(num3, num4)
```
Когда создается функция необходимо очень аккуратно давать наименование параметрам. Например, здесь мне мне понятно почему именно num3 и num4, почему не num1/num2

Reply:
Я почему-то думала что это продолжение предыдущего задания с lambda-выражением и начала именовать переменные по порядку. Потом поняла что нужно делать, а переименовать переменные забыла...